.github/pull_request_template.md

```diff
@@ -0,0 +1,20 @@
+# Brief Description
```

Comment:
Also, should we remove this section?
This is basically what the PR title should be (and the PR title becomes the first line of the commit message anyways)

Reply:
I'm on board with that.